lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -93,7 +93,7 @@ class FeatureCard extends React.Component {
 
   componentWillUnmount() {
     if (this.requests) {
-      for (var request in this.requests) {
+      for (let request of this.requests) {
```

Comment:
You could also do this with the array's forEach method: `this.requests.forEach(request => request.abort());`